gemini/src/main/java/com/techempower/gemini/cluster/jms/CacheMessageManager.java

```diff
@@ -484,19 +472,10 @@ else if (group instanceof EntityGroup)
               // sent over the message queue. But *this* instance does not have
               // it as a CacheGroup, only an EntityGroup, which means we have
               // nothing to update here.
-              log.debug("Received 'cache object expired' for an " +
-                  "EntityGroup, so not cached. Ignoring:{}", cacheMessage);
             }
-            else
-            {
-              log.info("Receiving 'cache object expired' but group id is " +
-                  "invalid:{}, group: {}", cacheMessage, group);
-              application.getNotifier().addNotification(
```

Comment:
Why was this removed? I think some applications use this.

Reply:
Hm. Weird. Seems like git's auto-resolve thing with the merge conflict felt like removing this. Will add it back and re-review what it removed.

Reply:
Actually, this wasn't part of the conflict. Apparently I removed this. What?

Reply:
Oh, wait, never mind, I didn't remove it. I was just confused by the line history I was looking at. Keith removed this line recently: https://github.com/TechEmpower/gemini/commit/9064e0aa99d85d8e07a98452f7bad6bbc0f6c007#diff-f840e62d66613d77a961f9408873d9a3L135-L140